src/style.css

```
@@ -0,0 +1,103 @@
+* {
+  box-sizing: border-box;
+}
+
+div {
+  height: 20px;
+}
+
+input {
+  outline: none;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  margin-top: 0;
+}
+
+form {
+  position: relative;
+}
+
+.form-field {
+  display: block;
+}
+
+.form-field-big input {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 70px;
+  padding: 26px 62px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form-field-small input {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 42px;
+  padding: 12px 33px 14px;
+  font-size: 14px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form-field-big::after {
+  content: "";
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 19px;
+  display: inline-block;
+  position: absolute;
+  left: 25px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.form-field-small::after {
+  content: "";
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 11px;
+  display: inline-block;
+  position: absolute;
+  left: 12px;
+  top: 15px;
+  width: 11px;
+  height: 11px;
+}
+
+::placeholder {
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #d8d8e0 100%);
```

```suggestion
  background: linear-gradient(180deg, #fff 0, #f6f6f7 100%);
```
be attentive to design

---

Yes, I forgot to change(Thanks.